project-fortis-interfaces/src/components/Admin/Streams/StreamConstants.js

```diff
@@ -0,0 +1,372 @@
+import values from './../../../../node_modules/lodash/values';

```

Comment:
Same as above: please check if you can import this as `import values from 'lodash/values'`